public/uploadStudentList.php

```diff
@@ -0,0 +1,112 @@
+
+<?php 
+require_once "../modules/models/user.php";
+require_once "../common.php";
+
+    if (isset($_POST['submit2'])) 
+	{
+        
+		$students = fopen('studentList.csv', 'r');
+        $count = 0;
+		while (($studentData = fgetcsv($students, 1000, ",")) !== FALSE) 
+        {
+                $user = new User();
+                $user->student_id = $studentData[0];
+                $user->first_name = $studentData[1];
+                $user->last_name = $studentData[2];
+                $user->email = $studentData[3];
+                $user->is_admin = 0;
+                $user->is_instructor = 0;
+                $user->password_digest = password_hash("welcome", PASSWORD_DEFAULT);
+    
+                try {
+                    $user->save();
+                    $count++;
+                    $create_success = true;
+                }  catch(PDOException $error) {
+                     echo "<br>" . $error->getMessage();
+                }
+            
+    
+        }
+        
+        
+        fclose($students);
+        $filePath = 'studentList.csv';
+        unlink($filePath);
+
+	}
+?>
+
+<?php include "templates/header.php"; ?>
+
+<?php
+    if(isset($_POST['submit2']) && $create_success){
+        echo "<h3> $count Students added successfully</h3><br>";
```

Comment:
$create_success is potentially undefined here

Reply:
I think I fixed it